src/main/webapp/resources/js/pages/launch/launch-context.js

```diff
@@ -184,8 +178,16 @@ function LaunchProvider({ children }) {
           parameterSet: 0,
           shareResultsWithProjects: true,
           updateSamples: false,
+          emailPipelineResult: true,
```

Comment:
Same as above as it's a select now instead of a checkbox

Reply:
Will update in a separate PR